web/cobrands/greenwich/layout.scss

```diff
@@ -4,72 +4,139 @@
 
 #site-logo,
 body.frontpage #site-logo,
-body.twothirdswidthpage #site-logo
-{
-        position: relative;
-        top: 14px;
-        left: 14px;
-        margin: 0;
-        padding: 0;
-        background: url("/cobrands/greenwich/logo.gif") 0 0 no-repeat;
-        width: 170px;
-        height: 83px;
+body.twothirdswidthpage #site-logo {
+    position: relative;
+    top: 14px;
+    left: 14px;
+    margin: 0;
+    padding: 0;
+    background: url("/cobrands/greenwich/logo.gif") 0 0 no-repeat;
+    width: 170px;
+    height: 83px;
 }
 
 #front-main {
-    background-color: $base_bg;
+    background-color: $greenwich_light_grey;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
     padding-top: 0.5em;
     text-align: left;
 
     #front-main-container {
-        max-width: $container-max-width;
+        // max-width: $container-max-width;
         background-color: transparent;
         padding: 1em 0;
+        .intro-wrap {
+            max-width: 720px;
+        }
     }
 
     #postcodeForm div {
         margin: 0;
     }
 
     h1 {
-        color: $greenwich_dark_red;
+        // color: $greenwich_dark_red;
         padding-bottom: 3em;
     }
 }
 
 #site-header {
     background: none;
     padding: 0 1em; // "wave" background image means its easier to apply padding here than on .container
+    color: $contrast_text;
 
     .container {
         padding: 0; // let the #site-header padding do the work on mid-sized screens
     }
 
     .container:first-child {
-        background: url("/cobrands/greenwich/mastWave.gif") 0 0 no-repeat;
+        // background: url("/cobrands/greenwich/mastWave.gif") 0 0 no-repeat;
```

Comment:
You’ve removed the "wave" background image from the main site header here, but it still appears further down this page in `body.mappage #site-header`:

![Screenshot_2020-12-04 Hedge encroaching pavement - Viewing a problem](https://user-images.githubusercontent.com/739624/101146608-1afd0480-3613-11eb-8ded-0f15beda83b6.png)

Also, once you’ve removed it from there, remember to `git rm` the image file too.